### ISSUE TITLE
Change the data loading so that it can output a train set, a validation set and a test set

### ISSUE THREAD

**Issue by ValerianRey** [ValerianRey/fed_iot_guard#8]
For the autoencoder training, we need a train set, an opt set, a validation set and a test set.
